deployment/main.tf

```diff
@@ -0,0 +1,135 @@
+variable "gcp_project" {
+  type = string
+}
+
+variable "region" {
+  type    = string
+  default = "us-west1"
+}
+
+provider "google" {
+  project = var.gcp_project
+}
+
+provider "google-beta" {
+  project = var.gcp_project
+}
+
+# Enable required API in the project
+
+resource "google_project_service" "container-api" {
+  service = "container.googleapis.com"
+}
+
+resource "google_project_service" "spanner-api" {
+  service = "spanner.googleapis.com"
+}
+
+# Main cluster definition
+
+## Force recent version for Kubernetes master
+data "google_container_engine_versions" "gke-ver" {
+  location       = var.region
+  version_prefix = "1.14."
+}
+
+resource "google_container_cluster" "trillian-cluster" {
+  provider           = "google-beta"
```

Comment:
```
Warning: Quoted references are deprecated

  on main.tf line 37, in resource "google_container_cluster" "trillian-cluster":
  37: 	provider = "google-beta"

In this context, references are expected literally rather than in quotes.
Terraform 0.11 and earlier required quotes, but quoted references are now
deprecated and will be removed in a future version of Terraform. Remove the
quotes surrounding this reference to silence this warning.
```